dbms/src/Flash/Pipeline/Schedule/Tasks/Task.cpp

```diff
@@ -129,7 +139,7 @@ void Task::switchStatus(ExecTaskStatus to)
     }
 }
 
-void Task::assertStatus(ExecTaskStatus expect)
+void Task::assertNormalStatus(ExecTaskStatus expect)
 {
     RUNTIME_ASSERT(
```

Comment:
```suggestion
    RUNTIME_CHECK(
```

Reply:
I still prefer to use `RUNTIME_ASSERT`, because even if an error is thrown here, it will be caught and then terminate tiflash. 
As previously mentioned by https://github.com/pingcap/tiflash/pull/7438#discussion_r1188176813, I am unsure what would happen if only the exception is caught without terminating, such as incorrect query results being returned and so on.